Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -15,7 +19,7 @@ namespace Jellyfin.Drawing.Skia
     /// <summary>
     /// Image encoder that uses <see cref="SkiaSharp"/> to manipulate images.
     /// </summary>
-    public class SkiaEncoder : IImageEncoder
+    public class SkiaEncoder : CoreEncoder, IImageEncoder
```

Comment:
Should I move the CoreEncoder thing into an inner class of SkiaEncoder?

Reply:
Replace BlurHash.Net with this: https://www.nuget.org/packages/BlurHashSharp.SkiaSharp/
It has SkiaSharp support out of the box, and is faster ;)

Reply:
Ha, was gonna ask where you found that, but looking at the repo makes it pretty obvious! :)

Good call on using a lookup table for performance.

Reply:
Switched to `BlurHashSharp`.